src/components/NewWorkspaceModal.test.ts

```diff
@@ -147,6 +147,33 @@ describe('NewWorkspaceModal', () => {
     screen.getByText('You need a billing project to create a new workspace.');
   });
 
+  it('shows a message if there are no protected billing projects to use for creating a workspace with additional security monitoring ', async () => {
```

Comment:
I probably should update this to say "no GCP projects or Azure protected billing projects".